src/SCRIPTS/BF/PAGES/acc_calibrate.lua

```diff
@@ -0,0 +1,30 @@
+
+local template = assert(loadScript(radio.template))()
+local margin = template.margin
+local indent = template.indent
+local lineSpacing = template.lineSpacing
+local tableSpacing = template.tableSpacing
+local sp = template.listSpacing.field
+local yMinLim = radio.yMinLimit
+local x = margin
+local y = yMinLim - lineSpacing
+local inc = { x = function(val) x = x + val return x end, y = function(val) y = y + val return y end }
+local labels = {}
+local fields = {}
+
+if apiVersion >= 1.041 then
+	labels[#labels + 1] = { t = "Trim Accelerometer" ,     x = x,          y = inc.y(lineSpacing) }
+	fields[#fields + 1] = { t = "acc trim pitch",    x = x, y = inc.y(lineSpacing), sp = x + sp, min = -300, max = 300, vals = { 1 } }
+	fields[#fields + 1] = { t = "acc trim roll",     x = x, y = inc.y(lineSpacing), sp = x + sp, min = -300, max = 300, vals = { 2 } }
+end
+
+return {
+    read        = 240, -- MSP_ACC_TRIM
+    write       = 239, -- MSP_SET_ACC_TRIM         
+    title       = "ACC",
+    reboot      = true,
```

Comment:
Is a reboot needed for the trim to take effect?

Reply:
I don't know if it's needed. How can I test this?

Thx for all the other informations it's my fist commit,didnt know all the thinks only test it and it works 😉.

What happens if I do Wirte only 1 instead of 1, 2 due to 16 bit?

Reply:
"reboot = false". Then try making some changes, save the page and see if it took effect 🙂 

What happens is that the pitch field will modify the first byte of pitch trim and the roll field will modify the second byte of pitch trim. So the roll field will make unintended changes to pitch trim, and you won't be able to change roll trim at all